README.md

```diff
@@ -90,100 +84,13 @@ Date:   2015-12-11 17:57:10 +0100 +0100
 ...
 ```
 
-Retrieving the latest commit for a given repository:
-
-```go
-r, err := git.NewRepository("https://github.com/src-d/go-git", nil)
-if err != nil {
-	panic(err)
-}
-
-if err := r.PullDefault(); err != nil {
-	panic(err)
-}
-
-hash, err := r.Remotes[git.DefaultRemoteName].Head()
-if err != nil {
-	panic(err)
-}
-
-commit, err := r.Commit(hash)
-if err != nil {
-	panic(err)
-}
-
-fmt.Println(commit)
-```
-
-Creating a repository from an ordinary local git directory (that has been
-previously prepared by running `git gc` on it).
-
-```go
-// Download any git repository and prepare it as as follows:
-//
-//   $ git clone https://github.com/src-d/go-git /tmp/go-git
-//   $ pushd /tmp/go-git ; git gc ; popd
-//
-// Then, create a go-git repository from the local content
-// and print its commits as follows:
-
-package main
-
-import (
-	"fmt"
-	"io"
-
-	"gopkg.in/src-d/go-git.v4"
-	"gopkg.in/src-d/go-git.v4/utils/fs"
-)
-
-func main() {
-	fs := fs.NewOS() // a simple proxy for the local host filesystem
-	path := "/tmp/go-git/.git"
-
-	repo, err := git.NewRepositoryFromFS(fs, path)
-	if err != nil {
-		panic(err)
-	}
-
-	iter, err := repo.Commits()
-	if err != nil {
-		panic(err)
-	}
-	defer iter.Close()
-
-	for {
-		commit, err := iter.Next()
-		if err != nil {
-			if err == io.EOF {
-				break
-			}
-			panic(err)
-		}
-
-		fmt.Println(commit)
-	}
-}
-```
-
-Implementing your own filesystem will let you access repositories stored on
-remote services (e.g. amazon S3), see the
-[examples](https://github.com/src-d/go-git/tree/master/examples/fs_implementation/)
-directory for a simple filesystem implementation and usage.
-
-Wrapping
---------
-
-go-git can be wrapped into any language which supports shared library interop.
-[Python wrapper](https://github.com/src-d/gypogit) already exists.
-This is provided by "cshared" [cgo](https://golang.org/cmd/cgo/) files which can be built
-with `go build -o libgogit.so -buildmode=c-shared github.com/src-d/go-git/cshared`.
+Take note that the functions `CheckIfError` and `Info` are from the [examples package](https://github.com/src-d/go-git/blob/master/examples/common.go#L17) just to be used in the examples.
```

Comment:
Please note  that the functions `CheckIfError` and `Info` used in the examples were taken from the ...